### PR TITLE
[Raindrop] - Add and Delete bookmarks

### PR DESCRIPTION
**Pull request by ShubhamVerma1811**
I also wanted few QoL changes but didn't research much into the docs as of now.

- After a bookmark is added, it should take the user to "Latest bookmarks" page
- After a bookmark is deleted, it should get removed from the list.

Screenshot 2023-03-21 at 10 22 20
Screenshot 2023-03-21 at 10 21 55

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi @ShubhamVerma1811 🔥  Thanks for your contribution 💯 

@lardissone do you mind checking this 😊

**Review by lardissone — APPROVED** [review]
Awesome work! 
Just a small nit in a text :)

**Comment by ShubhamVerma1811** [comment]
Thank you for the review @lardissone!
And thanks for pushing the commit @pernielsentikaer!

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

* I don't see any way to submit this form
raindrop-io 2023-03-22 at 14 10 08

* When deleting a link there is no refresh of the list, you need to go out/in again before it's gone from the list

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by ShubhamVerma1811** [comment]
>  I don't see any way to submit this form
Weird, I am able to see it. Can it be some versioning issue?

Screenshot 2023-03-22 at 19 09 57

> When deleting a link there is no refresh of the list, you need to go out/in again before it's gone from the list
Is this something that needs to be implemented? I am not sure how it can be done. Couldn't find any Raycast docs on Navigating b/w command views..

**Comment by pernielsentikaer** [comment]
Hi 👋 

Check the new code from my PR, it's not done yet but fixed and enhanced multiple aspects of the extensions

- Added destructive style for `Delete Bookmark`
- Added hotkeys
- Added confirmAlert when deleting bookmark
- Fixed the add action, since the code was wrong
- Added validation for the add form
- Added more things

What is still left is the redraw of the list, I can maybe add this later today

Do you mind checking my corrections, feel free to modify 😊

File: extensions/raindrop/src/add.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            collectionId: values.collection,
+            tags: values.tags,
+          }),
+        }).then(async (res) => {
+          if (res.status === 200) {
+            toast.style = Toast.Style.Success;
+            toast.title = "Link Added";
+            toast.message = values.link;
+            reset({ link: "", collection: "-1", tags: [] });
+            console.log(await res.json());
```
Comment (lardissone) [inline extensions/raindrop/src/add.tsx:44]: nit: remove `console.log`

**Review by lardissone — APPROVED** [review]
Awesome work @pernielsentikaer ! Thank you!
Just left a small nit.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
We could indeed do this better with [optimisticUpdate](https://developers.raycast.com/utilities/react-hooks/useexec#mutation-and-optimistic-updates) but it works now.

Seems like there is a bug when you switch in the dropdown, could you look into that?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by ShubhamVerma1811** [comment]
Hey @pernielsentikaer, Are we waiting for any changes on this?

**Review by pernielsentikaer — APPROVED** [review]
Let's merge this one 🧑‍🚒

**Pull request merged**